dev/cognitiveservices/data-plane/Language/examples/questionanswering/authoring/SuccessfulProjectSubmitExportJob.json

```diff
@@ -13,6 +13,18 @@
     }
   },
   "responses": {
+    "200": {
```

Comment:
I fix the validation error by adding 200 into the example responses. Let me know if that is OK with you. @heaths , @sush2218